File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/exceptions/ErrorDisplayerActivity.java
package com.jdroid.android.sample.ui.exceptions;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class ErrorDisplayerActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return ErrorDisplayerFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/exceptions/ErrorDisplayerActivity.kt
package com.jdroid.android.sample.ui.exceptions

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class ErrorDisplayerActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return ErrorDisplayerFragment::class.java
    }
}
